llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1485,8 +1485,8 @@ static void transformRecipestoEVLRecipes(VPlan &Plan, VPValue &EVL) {
                     auto *CI = cast<CallInst>(CInst->getUnderlyingInstr());
```

Comment:
The TypeSwitch here has grown quite big, it would be good to outline it to a separate function to reduce the nesting level and make things slightly easier to read

Reply:
(would be good as. a follow up)

Reply:
OK~